sklearn/ensemble/_forest.py

```diff
@@ -632,6 +671,15 @@ def feature_importances_(self):
         all_importances = np.mean(all_importances, axis=0, dtype=np.float64)
         return all_importances / np.sum(all_importances)
 
+    def _more_tags(self):
+        # Ignore errors because the parameters are not validated
```

Comment:
Can you help me understand this a bit more? I was thinking that `_safe_tags` is the one that would raise an exception here, but as far as I can tell it would raise a `ValueError`. So I'm confused about where the exception comes from (`set_params`?) and why that means it is Ok to return an empty `{}` from `_more_tags`.